NEWS

```diff
@@ -4,6 +4,15 @@ Please send bug reports, questions and suggestions to
 <https://github.com/besser82/libxcrypt/issues>.
 
 Version 4.4.17
+* Salt string compatibility with generic implementations (issue #105).
+
+  It turned out, all other existing implementations of the md5crypt,
+  sha256crypt, and sha512crypt hashing methods allow any printable
+  ASCII character to be present in the salt string.
```

Comment:
FWIW, this wasn't really news (re: "turned out"), those other implementations allow non-ASCII and non-printable characters as well, but they might not allow '$'.

Reply:
Fixed in rebased commits.